Adyen.Test/HttpClientWrapperTest.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
```

Comment:
is this class used ?

Reply:
Sorry, I think I just forgot to delete this one. Made it for some experiments.

Reply:
Removed